indexer/packages/postgres/src/types/firebase-notification-token-types.ts

```diff
@@ -0,0 +1,24 @@
+/* ------- TOKEN TYPES ------- */
+
+type IsoString = string;
+
+export interface FirebaseNotificationTokenCreateObject {
+  token: string,
+  address: string,
+  language: string,
+  updatedAt: IsoString,
+}
+
+export interface FirebaseNotificationTokenUpdateObject {
+  token: string,
+  address: string,
+  language: string,
```

Comment:
do all these fields need to be present in the update object? if not, can we use `?:`